package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^7.2.1",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
+    "react-router-dom": "^5.1.2",
```

Comment:
Really cool using react router, even if not requested :)
You can't have a real SPA app without it.